p2p_wallet/Scenes/CreateOrRestoreWallet/Common/Pincode/Pincode/PincodeSnackbar.swift

```diff
@@ -1,4 +1,11 @@
 struct PincodeSnackbar {
+    let title: String?
     let message: String
     let isFailure: Bool
+
+    init(title: String? = nil, message: String, isFailure: Bool) {
+        self.title = title
+        self.message = message
+        self.isFailure = isFailure
```

Comment:
I removed the logic with this flag, do we still need it? can you check please?

Reply:
will check, thanks